wordpress-mobile/WordPress-iOS
# Issues/5162 tag display name

Fixes #5162 

Fixes the tag title and formatting to match what is in the web reader. 

To test: 
#### Test 1: Menu
- Open the reader menu and ensure tags are lower cased, and dashes instead of spaces.  Lists and default topics should remain capitalized and have spaces. 
- Follow a tag via the search field. Try following a tag like "Business & Technology" and ensure it follows as "business-technology".  Try following "business-technology" and ensure it follows correctly. 
- Unfollow a tag by tapping edit and deleting the tag. Ensure the tag is removed correctly.

#### Test 2: List and List Headers
- View tags shown in the reader post list cards. Ensure they match the new style
- Tap to preview a tag.  Ensure the header reflects the new formatting. 
- Follow a tag from the header. Open the menu and ensure it was followed. 
- Unfollow tag from the header. Open the menu and ensure it was unfollowed. 

Needs review: @frosty would you do the honors sir?
